source/packages/services/assetlibrary/infrastructure/cfn-assetLibrary.yaml

```diff
@@ -321,6 +350,25 @@ Resources:
           Value: !Ref CdfService
     DependsOn: LambdaFunction
 
+  # Optional Neptune auto-scaling configuration
+  AutoScalingNeptuneNestedStack:
+    Type: AWS::CloudFormation::Stack
```

Comment:
Open question: What's the benefit of having this be a nested stack instead of part of the top-level one?

Reply:
Just decided to follow what was set for Lambda autoscaling which was a nested stack.